app/src/main/java/com/hedvig/app/feature/keygear/KeyGearValuationActivity.kt

```diff
@@ -121,6 +130,14 @@ class KeyGearValuationActivity : BaseActivity(R.layout.activity_key_gear_valuati
         }
     }
 
+    private fun valuationType(item: KeyGearItemQuery.KeyGearItem): ValuationType {
+        return if (item.fragments.keyGearItemFragment.valuation is KeyGearItemFragment.AsKeyGearItemValuationFixed) {
```

Comment:
This is a little dangerous: Consider the scenario when we introduce a third valuation type. This code will incorrectly identify that third type as `MARKET_PRICE`. We need to consider a strategy that does not have the issue.